tests/Examples/Search/SearchPage.cs

```diff
@@ -1,68 +1,112 @@
 using Elastic.Xunit.XunitPlumbing;
+using Elasticsearch.Net;
+using Examples.Models;
 using Nest;
 
 namespace Examples.Search
 {
 	public class SearchPage : ExampleBase
 	{
-		[U(Skip = "Example not implemented")]
+		[U]
 		public void Line7()
 		{
 			// tag::9bdd3c0d47e60c8cfafc8109f9369922[]
-			var response0 = new SearchResponse<object>();
+			var searchResponse = client.Search<Tweet>(s => s
+				.Index("twitter")
+				.QueryOnQueryString("tag:wow")
+			);
 			// end::9bdd3c0d47e60c8cfafc8109f9369922[]
 
-			response0.MatchesExample(@"GET /twitter/_search?q=tag:wow");
+			searchResponse.MatchesExample(@"GET /twitter/_search?q=tag:wow", e =>
+			{
+				e.Method = HttpMethod.POST;
+				return e;
+			});
 		}
 
-		[U(Skip = "Example not implemented")]
+		[U]
 		public void Line340()
 		{
 			// tag::be49260e1b3496c4feac38c56ebb0669[]
-			var response0 = new SearchResponse<object>();
+			var searchResponse = client.Search<Tweet>(s => s
+				.Index("twitter")
+				.QueryOnQueryString("user:kimchy")
+			);
 			// end::be49260e1b3496c4feac38c56ebb0669[]
 
-			response0.MatchesExample(@"GET /twitter/_search?q=user:kimchy");
+			searchResponse.MatchesExample(@"GET /twitter/_search?q=user:kimchy", e =>
+			{
+				e.Method = HttpMethod.POST;
+				return e;
+			});
 		}
 
-		[U(Skip = "Example not implemented")]
+		[U]
 		public void Line386()
 		{
 			// tag::f5569945024b9d664828693705c27c1a[]
-			var response0 = new SearchResponse<object>();
+			var searchResponse = client.Search<Tweet>(s => s
+				.Index("kimchy,elasticsearch")
+				.QueryOnQueryString("user:kimchy")
+			);
 			// end::f5569945024b9d664828693705c27c1a[]
 
-			response0.MatchesExample(@"GET /kimchy,elasticsearch/_search?q=user:kimchy");
+			searchResponse.MatchesExample(@"GET /kimchy,elasticsearch/_search?q=user:kimchy", e =>
+			{
+				e.Method = HttpMethod.POST;
```

Comment:
Does the global change for this not get applied?

Reply:
Added a global handle for this.